__TESTS__/unit/actions/Effect.test.ts

```diff
@@ -260,4 +261,37 @@ describe('Tests for Transformation Action -- Effect', () => {
 
     expect(url).toBe('http://res.cloudinary.com/demo/image/upload/e_outline:outer:20:200/sample');
   });
+
+  it('Creates a cloudinaryURL with effect style_transfer', () => {
+    const url = new TransformableImage()
+      .setConfig(CONFIG_INSTANCE)
+      .effect(Effect.styleTransfer(image('woman')))
```

Comment:
Can a transformation be applied to the source for this effect? Looking at the code for `ImageSource` I think it should work, but we need to verify and add a test.

Reply:
Added as a test, the end result is, for example, 

```
l_woman/c_scale,h_100,w_100/e_style_transfer:preserve_color:15,fl_layer_apply/sample
```